assets/js/type-defs/cart.js

```diff
@@ -48,12 +48,13 @@
 /**
  * @typedef {Object} CartData
  *
- * @property {Array}      coupons       Coupons applied to cart.
- * @property {Array}      items         Items in the cart.
- * @property {number}     itemsCount    Number of items in the cart.
- * @property {number}     itemsWeight   Weight of items in the cart.
- * @property {boolean}    needsShipping True if the cart needs shipping.
- * @property {CartTotals} totals        Cart total amounts.
+ * @property {Array}      coupons               Coupons applied to cart.
+ * @property {Array}      selectedShippingRates array of selected shipping rates
+ * @property {Array}      items                 Items in the cart.
```

Comment:
Is this an array of shipping rate ids? If so, should be included in the description